File: Python_TP2/Codingan/Baju.py
from Aksesories import Aksesories

class Baju(Aksesories):
    def __init__(self, ID, Name, Category, Price, Stock, Size, Material, Brand, Factory="", Classification="", Type=""):
        super().__init__(ID, Name, Category, Price, Stock, Size, Material, Brand)
        self.Factory = Factory
        self.Classification = Classification
        self.Type = Type

    def setting_data_baju(self, ID, Name, Category, Price, Stock, Size, Material, Brand, NewFactory, NewClassification, NewType):
        super().setting_data_aksesories(ID, Name, Category, Price, Stock, Size, Material, Brand)
        self.Factory = NewFactory
        self.Classification = NewClassification
        self.Type = NewType

    def getter_factory(self):
        return self.Factory

    def getter_classification(self):
        return self.Classification

    def getter_type(self):
        return self.Type

File: Java_TP2/Codingan/Baju.java
public class Baju extends Aksesories {
    private String Factory;
    private String Classification;
    private String Type;

    public Baju() {
        super();
        this.Factory = "";
        this.Classification = "";
        this.Type = "";
    }

    public Baju(int ID, String Name, String Category, int Price, int Stock, String Size, String Material, String Brand, String Factory, String Classification, String Type) {
        super(ID, Name, Category, Price, Stock, Size, Material, Brand);
        this.Factory = Factory;
        this.Classification = Classification;
        this.Type = Type;
    }

    public void settingDataBaju(int ID, String Name, String Category, int Price, int Stock, String Size, String Material, String Brand, String NewFactory, String NewClassification, String NewType) {
        super.settingDataAksesories(ID, Name, Category, Price, Stock, Size, Material, Brand);
        this.Factory = NewFactory;
        this.Classification = NewClassification;
        this.Type = NewType;
    }

    public String getterFactory() {
        return this.Factory;
    }

    public String getterClassification() {
        return this.Classification;
    }

    public String getterType() {
        return this.Type;
    }
}